imports/node-app/core/createApolloServer.js

```diff
@@ -31,6 +32,10 @@ export default function createApolloServer(options = {}) {
   const schemas = options.schemas || [];
   const schemasToMerge = schemas.filter((td) => typeof td !== "string");
   const typeDefs = schemas.filter((td) => typeof td === "string");
+  const env = envalid.cleanEnv(process.env, {
+    GRAPHQL_INTROSPECTION_ENABLED: bool({ default: false }),
+    GRAPHQL_PLAYGROUND_ENABLED: bool({ default: false })
+  });
```

Comment:
Please add a new file `/imports/node-app/core/config.js` and move the `cleanEnv` call plus the `envalid` imports into that file. Export the cleaned config from there.

Then do `import config from "./config";` in this file and `config.GRAPHQL_INTROSPECTION_ENABLED`, etc.

This pattern was decided recently but not yet added to the Reaction public docs so you could not have known.

Reply:
Will do.

Reply:
Fixed